src/ngraph/runtime/gpu/gpu_cuda_kernel_builder.cpp

```diff
@@ -23,15 +23,15 @@ using namespace ngraph;
 void runtime::gpu::CudaKernelBuilder::get_elementwise_op(codegen::CodeWriter& writer,
                                                          const std::string& name,
                                                          const std::string& op,
-                                                         const std::vector<std::string>& data_types)
+                                                         const std::vector<std::string>& dtypes)
```

Comment:
It would have been better to do the rename in a separate PR. This just adds noise.

Reply:
`daya_types` should be good?

Reply:
It just needs to be consistent, `dtypes` were used in some places while `data_types` were used elsewhere. I chose dtypes because it is shorter and more commonly seen in other frameworks (mxnet, neon, tf, etc). I can remove it from this PR if desired.